iStore/Cart/CartViewModel.swift

```diff
@@ -19,9 +19,9 @@ final class CartViewModel: ObservableObject {
         self.cartManager = cartManager
     }
     
-    func loadUserCarts() {
+    func loadUserCarts(withUserId userID: Int) {
         
-        let url = userCartURL + "1"
+        let url = userCartURL + "\(userID)"
```

Comment:
I am seeing sometimes you are adding new line after as a first line in the function and sometimes not. Please follow the same structure for all functions